float64 set_point
float64 position
float64 velocity
float64 torque
float64 torque_measured
float64 time_step
int32   count